app/views/items/show.html.erb

```diff
@@ -0,0 +1,43 @@
+<div class="container new-item-form">
+  <div class="col-xs-12 col-md-6">
+    <div class="title-price"> <strong>Name: </strong><%= @presenter.name %> </div>
+    <p> <strong> Description:</strong>  <%= @presenter.description %></p>
+    <p> <strong>Buy It Now Price: $<%= @presenter.buy_it_now_price %></strong> </p>
+    <%= button_to "Buy Now", create_purchase_path(item_id: @presenter.id), class: "btn btn-xs btn-buy-now", id: "buy_now_button" %>
+  </div>
+  <div class="col-xs-12 col-md-6">
+    <p> <strong>Starting Bid:</strong> $<%= @presenter.starting_bid_price %> </p>
+    <p> <strong>Current Bid: </strong>$<%= @presenter.highest_bid %> </p>
+    <%= form_tag create_bid_path(item_id: @presenter.id), method: :post, :class => "form-inline", id: "bid_now_form" do %>
+      <%= text_field_tag :amount %>
+      <%= submit_tag "Place Bid", class: "btn btn-default, btn-buy-now"%>
+    <% end %><br>
+    <div id="countdown">
+      <p> Auction Details: <span id="auction_end_time"> <%= @presenter.auction_end_time %> </span></p>
+      <% if @presenter.auction_ongoing %>
+        <p> Auction ends in:
+        <span id="days"> <%= @presenter.days %> </span> days
+        <span id="hours"><%=  @presenter.hours %> </span> hours
+        <span id="minutes"> <%= @presenter.minutes %> </span> minutes
+        <span id="seconds"> <%= @presenter.seconds %> </span> seconds
+      <% end %>
+        </p>
+    </div>
+  </div>
+  <% if @presenter.errors %>
+    <ul>
+      <% @presenter.errors.each do |error| %>
+        <li> <%= error %> </li>
+      <% end %>
+    </ul>
+  <% end %>
+</div>
+<script>
+updateCountdown();
```

Comment:
So far, all you have is a very procedural function to call. It's very specific to the elements on this one page, which means it can't be reused, and also doesn't fully encapsulate it's responsibility because you still had to use setInterval on this page. I'd prefer to see a more object-oriented approach with hard dependencies (your html elements) injected into the function rather than referenced from within. Something like this:
```js
var trackingElements = {
  days: $("#days"),
  hours: $("#hours"),
  minutes: $("#minutes"),
  seconds: $("#seconds")
}
var countdown = new Countdown(endTime, trackingElements)
countdown.onEnd(function() {
    $("#auction_end_time").html("Auction for this item is closed");
    $("#buy_now_button").hide();
    $("#bid_now_form").hide();
})
countdown.start()
```